en/docs/develop/instagram-authenticator.md

```diff
@@ -1,5 +1,10 @@
 # Configuring Instagram Authenticator
 
+!!! note
+        This authenticator is now deprecated as the Instagram API 
```

Comment:
Too much indentation. This line should be indented to the level where the word "note" starts for the macro to appear correctly.